spec/rspec/core/world_spec.rb

```diff
@@ -224,6 +224,15 @@ module RSpec::Core
             world.announce_filters
           end
         end
+
+        context "with a filter but with silence_filter_announcements" do
+          it "does not announce" do
+            allow(configuration).to receive(:silence_filter_announcements?) { true }
```

Comment:
I'd prefer to see this set `configuration.silence_filter_announcements = true`.  Stubbing it doesn't have any advantages over setting it as far as I can see.